_rules/scrollable-element-keyboard-accessible-0ssw9k.md

```diff
@@ -226,7 +228,7 @@ This [scrollable][] `section` element has no [visible][] content.
 This `section` element has a [horizontal scroll distance][scrollable] that is less than its horizontal [padding][], and [vertical scroll distance][scrollable] that is less than its vertical [padding][].
 
 ```html
-<section style="height: 210px; width: 500px; overflow: scroll; padding: 30px;">
+<section style="height: 210px; width: 530px; overflow: scroll; padding: 30px 0 30px 30px;">
 	<div role="heading" aria-level="1">WCAG 2.1 Abstract</div>
 	<div style="width: 520px">
```

Comment:
I'm not sure the example illustrates the same thing now.
The old example is great at seeing if tools take `padding` into consideration: without `padding` it looks like a `520px` element inside a `500px` one and requires scrolling, with padding the inner element is less wide. No, the inner element is always less wide than the outer, and the example doesn't differentiate tools on whether they use `padding` or not.